server/utils/referrals/showRisksAndNeedsUtils.ts

```diff
@@ -13,11 +13,23 @@ export default class ShowRisksAndNeedsUtils {
         href: paths.show.risksAndNeeds.offenceAnalysis({ referralId }),
         text: 'Section 2 - Offence analysis',
       },
+      {
+        href: paths.show.risksAndNeeds.roshAnalysis({ referralId }),
+        text: 'Section R6 - ROSH analysis',
+      },
     ]
 
     return navigationItems.map(item => ({
       ...item,
       active: currentPath === item.href,
     }))
   }
+
+  static textValue(value?: string): string {
```

Comment:
👍

Reply:
Just cherry-picked your commit here!

Reply:
I did wonder whether `showRisksAndNeedsUtils` should move to the `risksAndNeeds` rather than `referrals` folder at some point, but didn't bother here